templates/kubernetes/docs/1.18/charm-containerd.md

```diff
@@ -123,7 +123,7 @@ juju run-action containerd ACTION [parameters] [--wait]
       debug
     </h5>
   </div>
-  <div class="col-7">
+  <div class="col-5">
```

Comment:
@evilnick note I needed to tweak the col count here as the docs main content now has a total of 9. Would you mind updating the generation script please?